azure_pipelins/verify_msalcpp_per_pr_ios.yml

```diff
@@ -0,0 +1,61 @@
+# Starter pipeline
+# Start with a minimal pipeline that you can customize to build and deploy your code.
+# Add steps that build, run tests, deploy, and more:
+# https://aka.ms/yaml
+
+trigger: none # Non-PR branches
+pr:
+  branches:
+    include:
+    - '*'
+
+pool:
+  vmImage: 'macOS-10.15'
+
+resources:
+ repositories:
+   - repository: microsoft-authentication-library-for-cpp
+     type: github
+     endpoint: 'GitHub for AzureAD and Azure-Samples (as aadidgit service)'
+     name: AzureAD/microsoft-authentication-library-for-cpp
+     ref: develop
+
+variables:
+  runCodesignValidationInjection: false
+
+jobs:
+- job: iOS_OnPrCommit
+  displayName: MSAL CPP checks per commit
+  pool:
+    vmImage: macOS-10.15
+  timeoutInMinutes: 40
+  cancelTimeoutInMinutes: 1
+  workspace:
+    clean: all
+
+  steps:
+  - checkout: self
+    path: CommonCore
+  - checkout: microsoft-authentication-library-for-cpp
+    path: s
+
+# Update Subtree will be on hold as there is no way to take local in yet (1131656)
+  # - task: UsePythonVersion@0
+  #   displayName: 'Use Python 3.x'
+  # - task: PythonScript@0
+  #   displayName: Ensure generated Podspec for macOS is up to date
+  #   inputs:
+  #     scriptPath: scripts/update_msalobjc_subtree.py
+
+  - template: azure_pipelines/templates/ios-setup.yml@microsoft-authentication-library-for-cpp
+
+  - template: azure_pipelines/templates/build-and-test.yml@microsoft-authentication-library-for-cpp
+    parameters:
+      configuration: Debug
+      platform: iOS
+      arch: x64
```

Comment:
Just curious - why do we not need to also provide arch support for arm?

Reply:
Could you clarify the question? What is arm here?